src/PiggyCustomEnchants/Entities/PigProjectile.php

```diff
@@ -99,7 +99,7 @@ protected function sendSpawnPacket(Player $player): void
         $pk->entityRuntimeId = $this->getId();
         $pk->position = $this->asVector3();
         $pk->motion = $this->getMotion();
-        $pk->metadata = $this->dataProperties;
+        $pk->metadata = $this->getDataPropertyManager()->getAll();
```

Comment:
Change to $this->propertyManager->getAll() to be more like the parent function.

Reply:
Changed as requested here:
https://github.com/95CivicSi/PiggyCustomEnchants/commit/275a32abf87202c11202e60dabf9837df753b0a7